src/__mocks__/react-i18next.js

```diff
@@ -4,15 +4,14 @@ import * as React from 'react'
 import i18next from 'i18next'
 
 // this mock makes sure any components using the translate HoC receive the t function as a prop
-export const withNamespaces = <Props> (namespace: string) => (Component: React.ComponentType<Props>) => {
-  const Translated = (props: Props) => <Component t={key => `${namespace}:${key}`} {...props} />
-  Translated.displayName = `Translate(${Component.displayName || Component.name})`
+const withTranslation = <Props> (namespace: string) => (Component: React.ComponentType<Props>) => {
+  const i18n = i18next.createInstance()
```

Comment:
Why do we need an i18next instance in here? I think it is only used in the tests and expected to be anything.